src/styles/foundation/_focus-ring.scss

```diff
@@ -1,12 +1,17 @@
 /// Sets the focus ring for an interactive element
 /// @param {String} $size - The size of the border radius on the focus ring.
-/// @param {String} $offset - Focus ring offset border width + 1px.
+/// @param {String} $border-width - The border width of your element in rems.
 /// @param {String} $style - Focus ring state.
 ///
 
-@mixin focus-ring($size: 'base', $offset: rem(1px), $style: 'base') {
+@mixin focus-ring($size: 'base', $border-width: rem(0), $style: 'base') {
   $stroke: rem(2px);
-  $position: calc(#{$offset} + #{$stroke});
+  $offset: calc(#{$border-width} + #{rem(1px)});
```

Comment:
Adding `calc` here allows us to pass in custom properties but by definition we prefer a rem value.